lib/src/screens/initial_setup/gen_file_body.dart

```diff
@@ -157,25 +156,28 @@ String genPubKeyFileBody(String resUrl, String pubKeyStr) {
   return keyFileBody;
 }
 
-/// Generates a profile file body in a predefined format.
-///
-/// This function takes two maps, `profData` and `authData`, as inputs. The `profData`
-/// map contains user profile information such as name and gender. The `authData` map
-/// includes authentication information, specifically an access token.
+// comment out the following function as it is not used in the current version
+// of the app, anushka might need to use to in the future so keeping it here.
```

Comment:
Hi @anushkavidanage , could you please have a look at the following part, if it is not needed, I can delete them. If we need to use them in the future, we can comment out just like what I did. :)